frontend/editor.css

```diff
@@ -649,6 +649,7 @@ nav#at_the_top > div.desktop_picker span {
 
 pluto-filepicker .cm-scroller {
     scrollbar-width: none; /* Firefox */
+    overflow-x: hidden;
```

Comment:
This setting must be moved to `pluto-input .cm-editor .cm-scroller`.
Also, the value must be `visible`, not `hidden`.

Reply:
In my case, I added the following lines near line 2789 of this file and the scrollbar disappeared.
```css
pluto-input .cm-editor .cm-scroller {
    overflow-x: visible;
}
```

Reply:
@holomorphism I updated the PR with `overflow-x: hidden` with the right cm instance selector. Have you tried with `overflow-y: hidden` on the same selector (since the scrollbars are vertical) ?

Reply:
@Pangoraw Sorry, you are right. When I set `overflow-y: hidden` instead of `overflow-x: visible`, the vertical scrollbar disappeared as well. (Note that scrollbars are still visible with the current `overflow-x: hidden`)

I think it would be appropriate to simply write only `overflow-y: hidden` there. Could you please fix this?

After checking with DevTools in chrome and finding `.cm-scroller` set to `overflow-x: auto`, I changed this value to `visible` and the vertical scrollbars disappeared, so I was happy with that.
I don't remember why I didn't try `overflow-y: hidden`, but I'm a little embarrassed!

Reply:
No problems, thanks for the extensive investigation!